pkg/ui/src/views/reports/containers/stores/index.tsx

```diff
@@ -58,9 +56,9 @@ class Stores extends React.Component<StoresProps, {}> {
     );
   }
 
-  renderStore(store: protos.cockroach.server.serverpb.StoreDetails$Properties, key: number) {
```

Comment:
would be good to use the `Loading` component in this file. See `jobs/index.tsx` for an example.